laos_climate_change/data_loaders.py

```diff
@@ -0,0 +1,29 @@
+import os
+from os.path import exists
+from urllib.request import urlretrieve
+
+world_url = "https://datacatalogfiles.worldbank.org/ddh-published/0038272/DR0046659/wb_countries_admin0_10m.zip?versionId=2024-05-14T14:58:01.5696428Z"
+world_filename = "wb_countries_admin0_10m.zip"
+laos_url = "https://data.humdata.org/dataset/9eb6aff1-9e3f-43d3-99a6-f415fe4b4dff/resource/1d6edf99-5303-4b31-8909-dd70cda78443/download/lao_adm_ngd_20191112_shp.zip"
+laos_filename = "lao_adm_ngd_20191112_shp.zip"
+
+
+def download_shapefile(
+    which, output_path=".../laos-climate-change/laos_climate_change"
```

Comment:
only 2 dots in the beginning

Reply:
> only 2 dots in the beginning

Great catch!

I just fixed it and pushed the new version